### PR TITLE
updated docs to reflect the 0.3.0 release

### PR DESCRIPTION
**Pull request by gallacher**
# Description

Updates to the documentation to reflect changes for the Observability v0.3.0 release.

# Issues

List the issues impacted by this PR:

| Issue ID |
| -------- |
|  #28         |

# Checklist:

- [x] I have performed a self-review of my own changes.

### CODE REVIEW

**Comment by ioeicicdsvc** [comment]
@gallacher: Thank you for submitting your pull request!

Please make sure you have followed our contributing guidelines. We will review it as soon as possible.

------------------------------------------------------------------------------------------------------------------------
We want your feedback! If you have any questions or suggestions regarding our contributing process/workflow, please reach out to us at karavi@dell.com.

File: installer/README.md [hunk 1]
```diff
@@ -245,15 +245,14 @@ or
...
-> namespace where Karavi Observability is to be installed.
->
-> Example command to copy the secret from the vxflexos namespace to the karavi namespace.
->```
->[user@anothersystem /home/user/offline-karavi-observability-bundle/helm]# kubectl get secret vxflexos-config -n vxflexos -o yaml | sed 's/namespace: vxflexos/namespace: karavi/' | kubectl create -f -
->```
+3. The vxflexos-config Secret from the namespace where CSI Driver for Dell EMC PowerFlex is installed must be copied to the namespace where Karavi Observability is to be installed.
 
-3. Now that the required images have been made available and the Helm chart's configuration updated with references to the internal registry location, installation can proceed by following the instructions that are documented within the Helm chart's repository.
+Example command to copy the secret from the vxflexos namespace to the karavi namespace.
```
Comment (hoppea2) [inline installer/README.md:250]: Consistency: Secret vs secret

File: docs/GETTING_STARTED_GUIDE.md [hunk 2]
```diff
@@ -87,16 +84,14 @@ Before installing the karavi-observability chart, you must first install the cer
...
->
-> Copy the vxflexos-config Secret from the CSI Driver for Dell EMC PowerFlex namespace into the namespace where Karavi Observability is deployed.
-> Note: The target namespace must exist before executing this command.
->
-> *Example command to copy the secret from the vxflexos namespace to the karavi namespace.*
->
-> ```console
-> $ kubectl get secret vxflexos-config -n vxflexos -o yaml | sed 's/namespace: vxflexos/namespace: karavi/' | kubectl create -f -
-> ```
+Copy the vxflexos-config Secret from the CSI Driver for Dell EMC PowerFlex namespace into the namespace where Karavi Observability is deployed.
```
Comment (hoppea2) [inline docs/GETTING_STARTED_GUIDE.md:87]: Secret: We spell the other k8s components with lower case (e.g. "namespace")

File: installer/README.md (hunk 1 above)
Comment (gallacher) [reply installer/README.md]: Fixed

File: docs/GETTING_STARTED_GUIDE.md (hunk 2 above)
Comment (gallacher) [reply docs/GETTING_STARTED_GUIDE.md:87]: Fixed

**Review by tdawe — APPROVED** [review]

**Review by atye — APPROVED** [review]

**Pull request merged**